workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Worker/WorkerBaseTests.cs

```diff
@@ -1,59 +0,0 @@
-using Improbable.Gdk.TestUtils;
-using NUnit.Framework;
-using UnityEngine;
-
-namespace Improbable.Gdk.Core.EditmodeTests
-{
-    [TestFixture]
-    public class WorkerBaseTests
```

Comment:
Is there a reason for these to be removed? Shouldn't this functionality still exist?